Tasks/Maven/task.json

```diff
@@ -71,6 +71,19 @@
             "required": true,
             "groupName":"junitTestResults",
             "helpMarkDown": "Test results files path.  Wildcards can be used.  For example, `**/TEST-*.xml` for all xml files whose name starts with TEST-."
+        },
+        {
+            "name":"javaHomeSelection",
+            "type":"radio",
+            "label":"Set JAVA_HOME by",
+            "required":true,
+            "groupName":"advanced",
+            "defaultValue":"JDKVersion",
+            "helpMarkDown": "Sets JAVA_HOME either by selecting a JDK version or by enetering a path to the specific JDK manually.",
```

Comment:
This help markdown could be clarified with:
"Sets JAVA_HOME either by selecting a JDK version that will be discovered during builds or by manually entering a JDK path."